app/src/main/java/com/google/android/stardroid/activities/EditSettingsActivity.java

```diff
@@ -85,8 +85,10 @@ public void onStart() {
     super.onStart();
     analytics.trackPageView(Analytics.EDIT_SETTINGS_ACTIVITY);
     final Preference locationPreference = preferenceFragment.findPreference(LOCATION);
-    Preference latitudePreference = preferenceFragment.findPreference(LATITUDE);
-    Preference longitudePreference = preferenceFragment.findPreference(LONGITUDE);
+    final Preference latitudePreference = preferenceFragment.findPreference(LATITUDE);
+    final Preference longitudePreference = preferenceFragment.findPreference(LONGITUDE);
+    latitudePreference.setSummary(sharedPreferences.getString(LATITUDE, "Degrees"));
```

Comment:
Sorry, I can't accept changes that hard code strings - they need to be localized if they're user-visible.